projects/example-app-cypress/plugins/index.js

```diff
@@ -1,5 +1,7 @@
 const cypressTypeScriptPreprocessor = require('./cy-ts-preprocessor');
 
-module.exports = on => {
+module.exports = on => 
   on('file:preprocessor', cypressTypeScriptPreprocessor);
-};
+
+
+  require('@applitools/eyes.cypress')(module);
```

Comment:
- Remove the leading spaces before the `require`
- This should be `@applitools/eyes-cypress` instead of `@applitools/eyes.cypress`

Reply:
- spaces before require has been removed 

- @applitools/eyes.cypress is now @applitools/eyes-cypress

- [ ]

Reply:
ignore brackets